extensions/github-browser/extension-browser.webpack.config.js

```diff
@@ -21,10 +20,8 @@ module.exports = withDefaults({
 		alias: {
 			'node-fetch': path.resolve(__dirname, 'node_modules/node-fetch/browser.js'),
 		},
-	},
-	plugins: [
-		new webpack.DefinePlugin({
-			WEBWORKER: JSON.stringify(true)
-		})
-	]
+	}
 });
+config.plugins.push(new webpack.DefinePlugin({ WEBWORKER: JSON.stringify(true) }))
```

Comment:
@aeschli I think we should move this into the shared file -- so that if others need to have conditional code for node vs webworkers it is standardized.